packages/client/src/context/AuthContext.js

```diff
@@ -24,7 +24,8 @@ export const AuthContextProvider = ({ children }) => {
 
   const logout = useCallback(() => {
     _dispatch({});
-    localStorage.deleteItem("Token Object");
+    localStorage.removeItem("Token Object");
+    window.location.reload(false);
```

Comment:
This probably doesn't belong with this PR, it's fine for now. We probably want to do something like:
```js
const history = useHistory();

const logout = useCallback(() => {
  // ...
  history.push('/');
};
```